beginners-yoga-course.html

```diff
@@ -0,0 +1,79 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Prema Healing Cleveland</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Signika:300">
+  <link rel="stylesheet" href="main.css">
+  <style type="text/css"></style>
+</head>
+<body>
+<div class="pageWrap">
+
+<!-- HEADER -->
+<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
+  <header class="pageHeader">
+    <h1 class="headline pageHeader__headline align--center">
+      <span class="headline__line pageHeader__primary inverted">Prema Healing</span>
+      <span class="headline__line pageHeader__secondary">Cleveland</span>
+    </h1>
+  </header>
+</div>
+
+<div class="gridCell--row program__row gridCell--row--justifyCenter">
+
+  <!-- COL A -->
+  <div class="gridCell--col gridCell--padded program__col align--center">
+    <article class="program program-wide">
+
+      <h2 class="headline inverted uppercase program__title">Beginners Yoga Course</h2>
+
+      <div class="meta uppercase">
+        <p class="meta__item">
+          <span class="unbreakable">A five week course beginning April 13th – May 11th.</span>
+          <span class="unbreakable"></span>
+        </p>
+        <p class="meta__item">Wedensdays 6:30-7:30pm</p>
+      </div>
+
+      <div class="content">
+        <p>This course is a step-by-step introduction into Sivananda classical hatha yoga. Students will experience the Five Points of Yoga including proper exercise, breathing, diet, relaxation, and meditation. Sivananda yoga guides students through asanas, while relaxing between each posture. Classes begin with chanting, pranayama breathing, sun salutations and go into the 12 basic asanas. This course walks students through at their own pace and completion of the course will provide a full understanding of the Sivanada Open Class.</p><br>
+        <p><b>REGISTER BY APRIL 1ST TO SAVE YOUR SPOT!</b></p>
```

Comment:
WHY ARE YOU YELLING AT ME?

Challenge: can you capitalize this using only CSS? What are the pros/cons to doing so? 

Also I might suggest an `em` instead of a `b`. Minor thing, I don't feel like getting into details.